test/integration/targets/aws_acm_facts/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# tasks file for aws_acm_facts
+
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- block:
+    - name: Get all ACM certificate facts
+      aws_acm_facts:
+        <<: *aws_connection_info
+      register: result
+
+    - assert:
+        that:
+          - result.changed
```

Comment:
I don't expect a facts module to change things.

Reply:
Fixed. You are right of course.